### PR TITLE
Sidebar conversations

### PR DESCRIPTION
**Pull request by impguard**
Conversation control for the sidebar.

### CODE REVIEW

**Comment by whsieh** [comment]
I looks like you might have to replace a string or two in the invite spec.

File: app/assets/javascripts/homepage/sidebar.coffee [hunk 1]
```diff
@@ -20,30 +20,45 @@ sidebar.controller("SidebarController", ["$scope", "$http", "$state", "API", "Ti
...
+            .success (conversation) ->
+                $state.go("conversation", { id: conversation.id })
+                reloadSidebarArenas()
+            .error (result, status) ->
+                if result?
+                    reason = result.error
+                else
                 reason = "status code #{status}"
-            console.log "sidebar failed: #{reason}"
+                console.log "sidebar failed: #{reason}"
```
Comment (Fong-) [inline app/assets/javascripts/homepage/sidebar.coffee]: I'm being pedantic here (sorry) indentation on 32 and 33 is off

File: app/assets/javascripts/homepage/sidebar.coffee [hunk 2]
```diff
@@ -20,30 +20,45 @@ sidebar.controller("SidebarController", ["$scope", "$http", "$state", "API", "Ti
...
+    $scope.conversationStartClicked = (id) ->
+        API.createConversationByUser(id)
+            .success (conversation) ->
+                $state.go("conversation", { id: conversation.id })
+                reloadSidebarArenas()
+            .error (result, status) ->
+                if result?
+                    reason = result.error
+                else
                 reason = "status code #{status}"
```
Comment (nickfong) [inline app/assets/javascripts/homepage/sidebar.coffee]: I'm being pedantic here (sorry) indentation on line 32 is off

**Comment by nickfong** [comment]
LGTM +1

**Comment by whsieh** [comment]
+1

**Comment by nickfong** [comment]
Will merge momentarily once CI passes/I'm sure I didn't introduce any new bugs when rebasing

**Pull request merged**